app/src/main/kotlin/com/simplemobiletools/calculator/extensions/Context.kt

```diff
@@ -16,11 +19,19 @@ val Context.calculatorDB: CalculatorDao get() = CalculatorDatabase.getInstance(a
 fun Context.updateViewColors(viewGroup: ViewGroup, textColor: Int) {
     val cnt = viewGroup.childCount
     (0 until cnt).map { viewGroup.getChildAt(it) }
-            .forEach {
-                when (it) {
-                    is TextView -> it.setTextColor(textColor)
-                    is Button -> it.setTextColor(textColor)
-                    is ViewGroup -> updateViewColors(it, textColor)
-                }
+        .forEach {
+            when (it) {
+                is TextView -> it.setTextColor(textColor)
+                is Button -> it.setTextColor(textColor)
+                is ViewGroup -> updateViewColors(it, textColor)
             }
+        }
+}
+
+inline fun <reified T> Context.refreshAppWidget() {
```

Comment:
we have Context.updateWidgets() functions in some other apps including calendar, please use those, This solution looks strange.

Reply:
done!